src/Nethermind/Nethermind.Crypto/Nethermind.Crypto.csproj

```diff
@@ -44,12 +44,8 @@
 <!--        <Content Include="runtimes\win-x64\native\eth_pairings2537.dll">-->
 <!--            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>-->
 <!--        </Content>-->
-      <Reference Include="Microsoft.AspNetCore.DataProtection.Abstractions, Version=3.1.5.0, Culture=neutral, PublicKeyToken=adb9793829ddae60">
-        <HintPath>..\..\..\..\..\Program Files\dotnet\packs\Microsoft.AspNetCore.App.Ref\3.1.5\ref\netcoreapp3.1\Microsoft.AspNetCore.DataProtection.Abstractions.dll</HintPath>
-      </Reference>
-      <Reference Include="Microsoft.AspNetCore.DataProtection.Extensions, Version=3.1.5.0, Culture=neutral, PublicKeyToken=adb9793829ddae60">
-        <HintPath>..\..\..\..\..\Program Files\dotnet\packs\Microsoft.AspNetCore.App.Ref\3.1.5\ref\netcoreapp3.1\Microsoft.AspNetCore.DataProtection.Extensions.dll</HintPath>
-      </Reference>
+      <Reference Include="Microsoft.AspNetCore.DataProtection.Abstractions, Version=5.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60"/>
+      <Reference Include="Microsoft.AspNetCore.DataProtection.Extensions, Version=5.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60"/>
```

Comment:
I believe these were just missed in the update to .NET 5.  Also, the hint paths are specific to one person's machine so I removed them.

Reply:
there is some mess with this on various machines building and failing on it